integration/querydsl/expressions/src/main/java/com/blazebit/persistence/querydsl/AbstractBlazeJPAQuery.java

```diff
@@ -78,7 +79,7 @@
 
     protected final CriteriaBuilderFactory criteriaBuilderFactory;
 
-    protected boolean cachable = false;
+    protected boolean cacheable = false;
```

Comment:
Fix typo, unlikely to affect users because accessors where typed correctly and users ought not to subclass `BlazeJPAQuery`.